app/views/labware/_basic_relative.html.erb

```diff
@@ -0,0 +1,11 @@
+<%# A simple representation of any labware type not currently handled otherwise -%>
+<%# locals: (labware:, open_in_new_window: false) -%>
+<%
+  labware_path = limber_plate_path(labware.uuid) + "#relatives"
+%>
+<div class="relative-item">
+    <%= labware.purpose.name %> - <%= link_to labware_path, class: 'card-link stretched-link', target: (open_in_new_window ? '_blank' : '_self') do %>
+        <%= labware.barcode.human %>
```

Comment:
I am not sure all labware have a human barcode, probably fine as long as this doesn't 500 in that case.